lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -2249,8 +2257,10 @@ DEFINE_ISXLOCALEAVAILABLE(PR, uloc)
             // TODO(jahorto): Determine if this would ever be returned and what it would map to
             case UNUM_PERMILL_FIELD: AssertOrFailFastMsg(false, "Unexpected permill field");
 
-            case UNUM_SIGN_FIELD: num < 0 ? library->GetIntlMinusSignPartString() : library->GetIntlPlusSignPartString();
-            default: AssertOrFailFastMsg(false, "Unexpected unknown part"); return nullptr;
+            case UNUM_SIGN_FIELD: return num < 0 ? library->GetIntlMinusSignPartString() : library->GetIntlPlusSignPartString();
+
+            // At the ECMA-402 TC39 call for May 2017, it was decided that we should treat unmapped parts as type: "unknown"
```

Comment:
>// At the ECMA-402 TC39 call for May 2017, it was decided that we should treat unmapped parts as type: "unknown" [](start = 12, length = 112)

oh, does this comment also apply to the other place I asked about UnkonwnPartString?

Reply:
Correct